src/MLJMultivariateStatsInterface.jl

```diff
@@ -150,4 +150,1078 @@ metadata_pkg.(
+multivariate Gaussian class-conditional distribution
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+
+    mach = machine(model, X)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitypes with `schema(X)`
+- `y`: is the target, which can be any `AbstractVector` whose element
+  scitype is `<:OrderedFactor(2)` or `<:Multiclass(2)`; check the scitype
+  with `scitype(y)`
+
+Train the machine using `fit!(mach, rows=...)`.
+
+# Hyper-parameters
+
+- `normalize=true`: Option to normalize the between class variance for the number of
+   observations in each class, one of `true` or `false`.
+- `out_dim`: The dimension of the transformed space to be used by `predict` and
+   `transform` methods, automatically set if 0 is given (default).
+- `priors::Union{Nothing, Vector{Float64}}=nothing`: For use in prediction with Baye's
+    rule. If `priors = nothing` then `priors` are estimated from the class proportions
+    in the training data. Otherwise it requires a `Vector` containing class
+    probabilities with probabilities specified using the order given by `levels(y)`
+    where y is the target vector.
+
+
+# Operations
+
+- `transform(mach, Xnew)`: Return a lower dimentional projection of the input `Xnew` having the same scitype as `X` above.
+- `predict(mach, Xnew)`: Return predictions of the target given
+  features `Xnew` having the same scitype as `X` above. Predictions
+  are probabilistic.
+- `predict_mode(mach, Xnew)`: Return the modes of the probabilistic predictions
+   returned above.
+
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `projected_class_means`: The matrix comprised of class-specific means as
+  columns (of size `(d,m)`), where d corresponds to input features and m corresponds to class.
+- `projection_matrix`: The matrix used to project `X` into a lower dimensional space.
+- `priors`: The estimated class priors.
+
+# Report
+
+The fields of `report(mach)` are:
+
+- `explained_variance_ratio`: The ratio of explained variance to total variance. Each dimension corresponds to an eigenvalue.
+- `classes`: The classes seen during model fitting.
+- `class_means`: The matrix comprised of class-specific means as
+  columns (of size `(d,m)`), where d corresponds to input features and m corresponds to class.
+- `mean`: The mean vector.
+- `class_weights`: The weights of each class.
+- `nc`: The number of classes.
+
+# Examples
+
+```
+using MLJ
+
+bsLDA = @load BayesianSubspaceLDA pkg=MultivariateStats
+
+X, y = @load_iris
+
+model = bsLDA()
+mach = machine(model, X, y) |> fit!
+
+Xproj = transform(mach, X)
+y_hat = predict(mach, X)
+labels = predict_mode(mach, X)
+```
+
+See also
+[`LDA`](@ref), [`BayesianLDA`](@ref), [`SubspaceLDA`](@ref)
+"""
+BayesianSubspaceLDA
+"""
+$(MMI.doc_header(FactorAnalysis))
+
+`FactorAnalysis`(FA) is a linear-Gaussian latent variable model that is
```

Comment:
This "factor analysis" doc string is sitting above "PPCA" method.

I'm going to pause my review here as these type-docstring interchanges are confusing.